### PR TITLE
Feature #13 authentication

### LINKED ISSUE #16: Login with Auth firebase module
# REQUIREMENTS
We need to implement a function to Log the users using Auth Firebase module.
The function will receive two parameters: email and password

# Acceptance Criteria
- [ ] The user can log on the app using their email and their password
- [ ] The app will show to loged user "Flights View", where they  can access to a list with all their reservations

### PR DESCRIPTION
**Pull request by HRuvalcabaP**
# Description
We created a function for log users with email and password.
We add a validation when the userData is wrong, the app show alert component.
 
 # Testing
 No.

##ScreenShot
image

### CODE REVIEW

**Review by Joaquin-Niembro — APPROVED** [review]

**Review by JordiGN — APPROVED** [review]
Cool, but I would like to see at least a screenshot of the alert you mentioned in the description, also, adding a little bit more information would be great

File: flightreservation/Views/Login.js [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
 const Login = ({navigation})=>
-{
+{  
+    
+  //-------------------------------------------------------------------------------//  
+  function logMailPassword(email,password)
+  {
+      auth().signInWithEmailAndPassword(email,password)
+      .then(() =>{
+          console.log("Logen succesfull whit", email)
```
Comment (develiasdaniel) [inline flightreservation/Views/Login.js:15]: mentors recommend removing console logs in PRs

File: flightreservation/Views/Login.js [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+                      { text: 'OK'}
+                    ],
+                    { cancelable: false }
+                  );                
+            }                
+        });                 
+  }
+  //----------------------------------------------------------------------------------------//
+  
+
```
Comment (develiasdaniel) [inline flightreservation/Views/Login.js:35]: mentors recommend that we use "prettier" from the visual code to accommodate the code

**Review by develiasdaniel — APPROVED** [review]
Excelent job! 🚀

**Review by JacintoOrtiz15745 — APPROVED** [review]

**Pull request merged**